core/src/test/scala/unit/kafka/coordinator/group/GroupMetadataManagerTest.scala

```diff
@@ -85,21 +82,47 @@ class GroupMetadataManagerTest {
       offsetsTopicCompressionCodec = config.offsetsTopicCompressionCodec,
       offsetCommitTimeoutMs = config.offsetCommitTimeoutMs,
       offsetCommitRequiredAcks = config.offsetCommitRequiredAcks)
+  }
 
-    defaultOffsetRetentionMs = offsetConfig.offsetsRetentionMs
-
+  private[this] def mockKafkaZkClient: KafkaZkClient = {
```

Comment:
Nit: we don't use `private[this]` for tests. It's mostly a micro-optimization is not particularly useful for tests (`private` provides good enough semantics).

Reply:
done